res/settings.ui

```diff
@@ -102,23 +89,17 @@
       <height>22</height>
      </rect>
     </property>
-    <property name="prefix">
-     <string/>
-    </property>
-    <property name="decimals">
-     <number>0</number>
+    <property name="cursor">
+     <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="minimum">
-     <double>30.000000000000000</double>
+     <number>20</number>
     </property>
     <property name="maximum">
-     <double>5000.000000000000000</double>
-    </property>
-    <property name="singleStep">
-     <double>1.000000000000000</double>
+     <number>240</number>
```

Comment:
Why these numbers make sense:
OOT runs at 20 visual frames during gameplay
No consumer monitor runs above 240hz, which is 0.00416ms, another option is to cap at 1000FPS for precision down to the milisecond. If anyone's computer can even realistically handle that.